test/CASpec.jl

```diff
@@ -0,0 +1,28 @@
+# ~/~ begin <<docs/src/carbocat-ca.md#test/CASpec.jl>>[init]
```

Comment:
@xyl96 This test makes sure that the new code gives the same results as the old.